testgen/src/commit.rs

```diff
@@ -119,9 +128,19 @@ impl Generator<block::Commit> for Commit {
                 signature: vote.signature,
             })
         };
-        let sigs = votes
+        let val_to_sig = |val: &Validator| -> Result<block::CommitSig, SimpleError> {
+            if let Some(vote) = votes
+                .iter()
+                .find(|&vote| vote.validator.as_ref().unwrap() == val)
```

Comment:
Can we extract this chained call into a variable for better readability?

Reply:
done